### DISCUSSION TITLE
VST"shell plugin" support?

### DISCUSSION THREAD

**Discussion by eikendrup** [apohl79/audiogridder#352] [General]
Does AG support VST shell plugins and can it route 16 audio channels I/O for ambisonics plugins ?

**Comment by apohl79** [comment]
It supports shell plugins like waves. Currently only 1/2 channels are supported.

**Comment by eikendrup** [comment]
Thank you for your very fast reply !

 
 

Gesendet: Mittwoch, 03. Februar 2021 um 12:56 Uhr
Von: "Andreas Pohl" <notifications@github.com>
An: "apohl79/audiogridder" <audiogridder@noreply.github.com>
Cc: "eikendrup" <abraring@gmx.de>, "Author" <author@noreply.github.com>
Betreff: Re: [apohl79/audiogridder] VST"shell plugin" support? (#352)


 

It supports shell plugins like waves. Currently only 1/2 channels are supported.
